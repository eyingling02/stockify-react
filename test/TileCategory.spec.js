import React from 'react'
import TileCategory from '../js/TileCategory.jsx'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

test('TileCategory snapshot test', () => {
  const component = shallow(<TileCategory />)
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
