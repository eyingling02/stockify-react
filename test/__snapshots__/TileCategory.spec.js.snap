exports[`test TileCategory snapshot test 1`] = `
<div
  className="outer">
  <div
    className="buy">
    <a
      href="javascript:;">
      BUY
    </a>
  </div>
  <div
    className="inner">
    <a
      href="livingroom.html"
      target="_blank">
      <h2 />
      <h2>
        %
      </h2>
      <h3>
         days
      </h3>
    </a>
  </div>
  <div
    className="sell">
    <a
      href="javascript:;">
      SELL
    </a>
  </div>
</div>
`;
